test/integration/default/controls/default_spec.rb

```diff
@@ -36,7 +36,7 @@
       it { should be_running }
     end
 
-    describe http('localhost') do
+    describe http('localhost', enable_remote_worker: true) do
```

Comment:
ChefDK 3 bundles InSpec 2.x where this is default behaviour.

Reply:
Actually yeah, I was playing with the tests. I can probably revert this again. Thanks